### PR TITLE
Ficus updates

### PR DESCRIPTION
**Pull request by nedbat**
## [OPEN-1867](https://openedx.atlassian.net/browse/OPEN-1867)

Changes to installation and release notes for Ficus.  Much of this will be cherry-picked onto master once everything has merged.

### Date Needed (optional)

If the release date of a feature is known or estimated, provide it to give reviewers guidance on turnaround time.

### Reviewers

Possible roles follow. The PR submitter checks the boxes after each reviewer finishes and gives :+1:. 

- [x] Subject matter expert: @gsong 
- [ ] Doc team review (sanity check, copy edit, or dev edit?): @edx/doc

FYI: Tag anyone else who might be interested in this PR here.

### Testing

- [ ] Ran ./run_tests.sh without warnings or errors

### HTML Version (optional)

- [ ] Build an RTD draft for your branch and add a link here

### Post-review

- [ ] Add a comment with the description of this change or link this PR to the next release notes task.
- [ ] Squash commits

### CODE REVIEW

**Comment by nedbat** [comment]
@jbarciauskas you might have opinions also.

**Comment by jbarciauskas** [comment]
Not me!

File: en_us/install_operations/source/installation/installation_options.rst [hunk 1]
```diff
@@ -5,7 +5,12 @@ Open edX Platform Installation Options
 ########################################
 
 This section describes Open edX installation options and the components that
-each option includes.
+each option includes. More details about the various options are at
+`Open edX Installation Options`_ page on the edX wiki.
+
+There are a number of Virtual Machine options, which install the Open edX
```
Comment (jbarciauskas) [inline en_us/install_operations/source/installation/installation_options.rst]: A number?

File: en_us/install_operations/source/installation/installation_options.rst [hunk 2]
```diff
@@ -20,15 +25,14 @@ stack (**fullstack**), or the Open edX analytics developer stack (**analytics
 devstack**).
 
 * Devstack is a Vagrant virtual machine instance designed for local
-  development. For more
-  information, see :ref:`Info Devstack`.
+  development. For more information, see :ref:`Info Devstack`.
 
 * Fullstack is a Vagrant virtual machine instance designed for deploying all
```
Comment (jbarciauskas) [inline en_us/install_operations/source/installation/installation_options.rst]: Are we replacing all instances of Fullstack with "fullstack/native"?

File: en_us/install_operations/source/installation/installation_options.rst [hunk 3]
```diff
@@ -20,15 +25,14 @@ stack (**fullstack**), or the Open edX analytics developer stack (**analytics
...
-  edX services on a single server. For more information, see :ref:`Info
+  Open edX services on a single server. For more information, see :ref:`Info
   Fullstack`.
```
Comment (jbarciauskas) [inline en_us/install_operations/source/installation/installation_options.rst]: fullstack

File: en_us/install_operations/source/installation/installation_options.rst [hunk 4]
```diff
@@ -46,47 +50,34 @@ Devstack simplifies certain production settings to make development more
...
+  Analytics development. For more information, see :ref:`Installing and
   Starting Analytics Devstack`.
 
 For more information about Vagrant, see the `Vagrant documentation`_.
 
 .. _Info Fullstack:
 
-====================
+==================
 Open edX Fullstack
```
Comment (jbarciauskas) [inline en_us/install_operations/source/installation/installation_options.rst:68]: Fullstack

File: en_us/install_operations/source/installation/installation_options.rst [hunk 5]
```diff
@@ -46,47 +50,34 @@ Devstack simplifies certain production settings to make development more
...
 
-Devstack is in the `edx configuration repository`_ on GitHub.
-
 For information about devstack and other installation and configuration options
-from edX and the Open edX community, see the `edx configuration repository
-wiki`_. Specifically, the following pages have more information about devstack.
-
-* `Devstack wiki`_
-* `Developing on Devstack`_
+from edX and the Open edX community, see the `Open edX Installation Options`_
```
Comment (jbarciauskas) [inline en_us/install_operations/source/installation/installation_options.rst:54]: Do we still need this link out?

File: en_us/install_operations/source/installation/installation_options.rst [hunk 6]
```diff
@@ -106,34 +97,45 @@ stack<Info Devstack>`. This development environment provides all of the
...
 
-Devstack also includes the following edX components.
+Devstack, fullstack and native installtion also include:
 
-* A demonstration edX course.
-* EdX Search.
+* E-Commerce
+* Programs
+* A demonstration Open edX course.
+* Open EdX Search.
```
Comment (jbarciauskas) [inline en_us/install_operations/source/installation/installation_options.rst]: Open edX

File: en_us/install_operations/source/installation/installation_options.rst [hunk 7]
```diff
@@ -178,14 +180,14 @@ Virtual Machine Configuration Options
...
-for edX virtual machines.
+for Open edX virtual machines.
 
 ================================================
 Set Up Ability to Preview Units (Mac/Linux Only)
 ================================================
 
-If you are installing an edX virtual machine on a Linux or Mac computer,
-you must configure your installation to use the preview feature in edX Studio.
+If you are installing an Open edX virtual machine on a Linux or Mac computer, you
```
Comment (jbarciauskas) [inline en_us/install_operations/source/installation/installation_options.rst:189]: Can you install an Open edX virtual machine on another OS?

File: en_us/install_operations/source/platform_releases/ficus.rst [hunk 8]
```diff
@@ -0,0 +1,100 @@
...
+***************************************
+Upgrading to a Subsequent Ficus Release
+***************************************
+
+Occasionally, we release updates to Ficus.  For example, the second 
+release of Ficus is ``open-release/ficus.2``.
+The steps to upgrade differ based on your original installation method.
+
+================================
+Upgrading a Vagrant Installation
```
Comment (jbarciauskas) [inline en_us/install_operations/source/platform_releases/ficus.rst:80]: I'm guessing from the formatting that it's clear that this is upgrading from ficus to ficus, but maybe worth clarifying?

File: en_us/open_edx_release_notes/source/ficus.rst [hunk 9]
```diff
@@ -0,0 +1,222 @@
...
+
+************************************************
+More Information on Ficus Release Changes
+************************************************
+
+The `edX Release Notes`_ contain a summary of changes that are deployed to
+edx.org. Those changes are part of the master branch of the edX Platform in
+GitHub.
+
+You can also find `release announcements`_ on the open.edx.org website. You can
```
Comment (jbarciauskas) [inline en_us/open_edx_release_notes/source/ficus.rst]: Should we encourage subscribing to this?

File: en_us/open_edx_release_notes/source/ficus.rst [hunk 10]
```diff
@@ -0,0 +1,222 @@
...
+
+    TBD
+
+
+************************************************
+More Information on Ficus Release Changes
+************************************************
+
+The `edX Release Notes`_ contain a summary of changes that are deployed to
+edx.org. Those changes are part of the master branch of the edX Platform in
```
Comment (jbarciauskas) [inline en_us/open_edx_release_notes/source/ficus.rst:196]: Add a note that these will eventually make it into a future Open edX release, though they will not be available before then unless you want to figure out how to upgrade your installation to the master branch (which is unsupported)?

**Review by jbarciauskas — CHANGES_REQUESTED** [review]

File: en_us/course_authors/source/conf.py [hunk 11]
```diff
@@ -5,7 +5,7 @@
 
 from shared.conf import *
 
-project = u'Building and Running an edX Course'
+project = u'Building and Running an edX Course: Ficus Release'
```
Comment (catong) [inline en_us/course_authors/source/conf.py]: @nedbat I think these Open edX project settings should be made to the Open edX version of the course authors conf.py instead?
en_us/open_edx_course_authors/source/conf.py

File: en_us/course_authors/source/index.rst [hunk 12]
```diff
@@ -5,9 +5,9 @@
 
 .. _document index:
 
-##################################
-Building and Running an edX Course
-##################################
+#################################################
+Building and Running an edX Course: Ficus Release
```
Comment (catong) [inline en_us/course_authors/source/index.rst:9]: Same comment as above - the Open edX named release changes should be made in the open_edx_course_authors source directory.

File: en_us/install_operations/source/installation/installation_options.rst (hunk 1 above)
Comment (nedbat) [reply en_us/install_operations/source/installation/installation_options.rst]: three :)  fixed.

File: en_us/install_operations/source/installation/installation_options.rst (hunk 2 above)
Comment (nedbat) [reply en_us/install_operations/source/installation/installation_options.rst]: I'm looking for the best wording for this.  The instructions and wiki pages still talk about fullstack.  I wanted these changes to amount to, "fullstack is a native install in a vagrant box." I guess that is confusing. I don't want to have to say "fullstack/native" everywhere.

File: en_us/install_operations/source/installation/installation_options.rst (hunk 5 above)
Comment (nedbat) [reply en_us/install_operations/source/installation/installation_options.rst:54]: I just added this link, because the old links were to outdated information.  The Installation Options wiki page explains how to choose among the options.  I suppose that should move into the rst docs, but I was trying to limit this pass to, "fix things that are wrong."

File: en_us/install_operations/source/installation/installation_options.rst (hunk 6 above)
Comment (nedbat) [reply en_us/install_operations/source/installation/installation_options.rst]: fixed.

File: en_us/install_operations/source/installation/installation_options.rst (hunk 7 above)
Comment (nedbat) [reply en_us/install_operations/source/installation/installation_options.rst:189]: Fullstack works fine on windows (I think).

File: en_us/install_operations/source/platform_releases/ficus.rst (hunk 8 above)
Comment (nedbat) [reply en_us/install_operations/source/platform_releases/ficus.rst:80]: Clarified

File: en_us/open_edx_release_notes/source/ficus.rst (hunk 10 above)
Comment (nedbat) [reply en_us/open_edx_release_notes/source/ficus.rst:196]: I added the second sentence here:
```
Changes listed for 11 January 2017 and before are included in the Ficus release
of Open edX.  Changes after that point will be in future Open edX releases.
```
I'd rather not give people encouragement to try to follow master.

File: en_us/open_edx_release_notes/source/ficus.rst (hunk 9 above)
Comment (nedbat) [reply en_us/open_edx_release_notes/source/ficus.rst]: It sounds like you mean, "maybe we should stop encouraging," since the words are already there.

File: en_us/open_edx_release_notes/source/ficus.rst (hunk 10 above)
Comment (jbarciauskas) [reply en_us/open_edx_release_notes/source/ficus.rst:196]: sgtm

File: en_us/open_edx_release_notes/source/ficus.rst (hunk 9 above)
Comment (jbarciauskas) [reply en_us/open_edx_release_notes/source/ficus.rst]: yes, sorry, didn't mean to be obtuse

File: en_us/course_authors/source/conf.py (hunk 11 above)
Comment (nedbat) [reply en_us/course_authors/source/conf.py]: oops! Fixed.

File: en_us/course_authors/source/index.rst (hunk 12 above)
Comment (nedbat) [reply en_us/course_authors/source/index.rst:9]: Fixed.

File: en_us/install_operations/source/installation/installation_options.rst (hunk 5 above)
Comment (jbarciauskas) [reply en_us/install_operations/source/installation/installation_options.rst:54]: That's fine, just checking

File: en_us/install_operations/source/installation/installation_options.rst (hunk 2 above)
Comment (jbarciauskas) [reply en_us/install_operations/source/installation/installation_options.rst]: I would just pick one (probably native) and treat fullstack as a special case of native.

File: en_us/open_edx_release_notes/source/ficus.rst [hunk 13]
```diff
@@ -0,0 +1,213 @@
...
+grades from peer assessments. For more information, see
+:ref:`opencoursestaff:Override a learner assessment grade`.
+
+Improvements to the word cloud tool include a new Instructions field and
+accessibility updates.
+
+The ability to prevent learners from receiving feedback until they've dropped
+all draggable items.
+
+Added "View this course as" capability.
```
Comment (catong) [inline en_us/open_edx_release_notes/source/ficus.rst]: I believe this description isn't complete - "View this course as" feature has existed for a while. If this item refers to TNL-6050, what got added in Ficus was the ability to view as a member of a content group (if cohorts and content groups exist in the course).

File: en_us/open_edx_release_notes/source/ficus.rst [hunk 14]
```diff
@@ -0,0 +1,213 @@
...
+ :depth: 1
+ :local:
+
+LMS
+===
+
+New **Next** and **Previous** buttons allow learners to navigate more
+intuitively through courses.
+
+A post listing view that shows not only post titles, but also the first line of
```
Comment (catong) [inline en_us/open_edx_release_notes/source/ficus.rst]: Might want to move this below or group with "Improvements to the discussions UI" (line 41)

File: en_us/open_edx_release_notes/source/ficus.rst [hunk 15]
```diff
@@ -0,0 +1,213 @@
...
+
+* :ref:`opencoursestaff:Multiple Responses in Numerical Input Problems` for
+  numerical input problems.
+
+* :ref:`opencoursestaff:Using the Script Element in Multiple Choice Problems`
+  for multiple choice problems.
+
+* :ref:`opencoursestaff:Using the Script Element in Checkbox Problems` for
+  checkbox problems.
+
```
Comment (catong) [inline en_us/open_edx_release_notes/source/ficus.rst]: Not sure about the intent for the links in these 2 bullet items (Randomized Content Blocks and Add Page)? They don't seem to make sense as part of the list about CAPA problems.

File: en_us/open_edx_release_notes/source/ficus.rst [hunk 16]
```diff
@@ -0,0 +1,213 @@
...
+* :ref:`opencoursestaff:Add Page`.
+
+In open response assessment (ORA) problems, course teams can override the
+grades from peer assessments. For more information, see
+:ref:`opencoursestaff:Override a learner assessment grade`.
+
+Improvements to the word cloud tool include a new Instructions field and
+accessibility updates.
+
+The ability to prevent learners from receiving feedback until they've dropped
```
Comment (catong) [inline en_us/open_edx_release_notes/source/ficus.rst]: Suggest rewording to refer to the problem type up front; also make active voice: "In drag and drop problems, learners no longer receive feedback until..."

File: en_us/open_edx_release_notes/source/ficus.rst (hunk 9 above)
Comment (nedbat) [reply en_us/open_edx_release_notes/source/ficus.rst]: Removed the mention of subscribing to release announcements.

File: en_us/open_edx_release_notes/source/ficus.rst (hunk 14 above)
Comment (nedbat) [reply en_us/open_edx_release_notes/source/ficus.rst]: Good idea, grouped.

File: en_us/install_operations/source/installation/installation_options.rst [hunk 17]
```diff
@@ -106,34 +97,45 @@ stack<Info Devstack>`. This development environment provides all of the
...
-**************************
 
-Fullstack, devstack, and analytics devstack all include the following edX
-components.
+*******************
+Native Installation
+*******************
+
+The Native installation installs the Open edX software on an Ubuntu 16.04
+machine.  Details are at the `Open edX Native Installation`_ page on the edX
```
Comment (gsong) [inline en_us/install_operations/source/installation/installation_options.rst]: Currently the link goes to 12.04 instructions. Will that be updated via the Confluence or do we need to change it here?

File: en_us/install_operations/source/platform_releases/ficus.rst [hunk 18]
```diff
@@ -0,0 +1,100 @@
...
+.. code-block:: bash
+
+    $ export OPENEDX_RELEASE=open-release/ficus.2
+    $ vagrant provision
+
+===============================
+Upgrading a Native Installation
+===============================
+
+If you installed Open edX using the `Open edX Native Installation`_, you can
```
Comment (gsong) [inline en_us/install_operations/source/platform_releases/ficus.rst:95]: Same thing here, currently link goes to 12.04 instructions.

File: en_us/open_edx_release_notes/source/ficus.rst (hunk 15 above)
Comment (nedbat) [reply en_us/open_edx_release_notes/source/ficus.rst]: I dug out the info, and made paragraphs for those two bullets.

File: en_us/open_edx_release_notes/source/ficus.rst (hunk 16 above)
Comment (nedbat) [reply en_us/open_edx_release_notes/source/ficus.rst]: Done

File: en_us/open_edx_release_notes/source/ficus.rst (hunk 13 above)
Comment (nedbat) [reply en_us/open_edx_release_notes/source/ficus.rst]: done.

**Comment by nedbat** [comment]
@catong @jbarciauskas @gsong Any other thoughts?

File: en_us/install_operations/source/installation/installation_options.rst (hunk 17 above)
Comment (nedbat) [reply en_us/install_operations/source/installation/installation_options.rst]: Right, I'll fix it.

**Comment by catong** [comment]
@nedbat Taking a last look, thanks

File: en_us/install_operations/source/platform_releases/ficus.rst (hunk 18 above)
Comment (nedbat) [reply en_us/install_operations/source/platform_releases/ficus.rst:95]: Done.

File: en_us/open_edx_release_notes/source/ficus.rst [hunk 19]
```diff
@@ -0,0 +1,218 @@
...
+######################
+
+This page lists the highlights of the Ficus release.
+
+.. contents::
+ :depth: 1
+ :local:
+
+**************
+New Features
```
Comment (catong) [inline en_us/open_edx_release_notes/source/ficus.rst]: It's a little strange to see "LMS", "Discussions" and "Studio..." listed as "New Features".
Suggestion: Change the "New Features" heading to something else, perhaps "Significant Updates". Then, also modify the intro on line 17 to something like: "The Open edX Ficus release includes the following updates."

File: en_us/open_edx_release_notes/source/ficus.rst [hunk 20]
```diff
@@ -0,0 +1,218 @@
...
+
+The following new features are included in the Open edX Ficus release.
+
+.. contents::
+ :depth: 1
+ :local:
+
+LMS
+===
+
```
Comment (catong) [inline en_us/open_edx_release_notes/source/ficus.rst:25]: In these sections, it might make for easier reading if each item were bulleted?

**Comment by catong** [comment]
@nedbat Made suggestions in comments, otherwise 👍

File: en_us/open_edx_release_notes/source/ficus.rst (hunk 19 above)
Comment (nedbat) [reply en_us/open_edx_release_notes/source/ficus.rst]: Hmm, I think I like it better as it is than labelling them "Highlights," but the other points I'll take :)

File: en_us/open_edx_release_notes/source/ficus.rst (hunk 20 above)
Comment (nedbat) [reply en_us/open_edx_release_notes/source/ficus.rst:25]: Done.

**Comment by nedbat** [comment]
@jbarciauskas I will accept your implicit assent :)

**Comment by jbarciauskas** [comment]
Yeah lots of others chimed in :)
